cmd/gnodev/util.go

```diff
@@ -104,3 +105,13 @@ func guessRootDir() string {
 	rootDir := strings.TrimSpace(string(out))
 	return rootDir
 }
+
+func makeTestGoMod() (string, error) {
+	var out bytes.Buffer
+
+	_, err := out.WriteString("module github.com/gnolang/gno\n\ngo 1.18\n")
+	if err != nil {
+		return "", fmt.Errorf("write to buffer: %w", err)
+	}
+	return out.String(), nil
+}
```

Comment:
I had to do a double take on this method - why are you using a buffer for creating the string, instead of a simple?

`return fmt.Sprintf(...)`

Reply:
I plan to extend it later. Or maybe support `gno.mod`. I'm not sure.

Reply:
Resolving this convo as you've rewritten the implementation